src/DigitalSubscriberAppBanner.tsx

```diff
@@ -11,9 +11,9 @@ import { PlayStore } from "./assets/play-store"
 const imgHeight = "300";
 
 export type Props = {
-  message: string;
   onButtonClick: (buttonType: string) => void;
-  firstName?: string;
+  header?: string;
+  body?: string;
```

Comment:
The `?` makes these props optional, but I think they should be required.

Reply:
I'd left them optional as they could be left blank in the Braze interface, which would perhaps yield a null? What do you think?

Reply:
Hmm good question. They both seem like required fields to me in that I don't think the banner makes sense without either of them.

Though it's true that these TypeScript types don't give us any kind of enforcement that they'll actually be set at runtime. If that's a concern then perhaps the right approach is to leave them optional but return early if either aren't set. I don't think we'd want to continue with the rendering of the banner if the body text or heading was missing. What do you reckon?

Reply:
Good idea, I've kept them optional but added a check that will check if they're empty.